instrumentation/netty/netty-4.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/netty/v4_1/ChannelInstrumentation.java

```diff
@@ -40,7 +39,7 @@
   @Override
   public void transform(TypeTransformer transformer) {
     transformer.applyAdviceToMethod(
-        isMethod().and(nameStartsWith("write")),
+        isMethod().and(named("write").or(named("writeAndFlush"))),
```

Comment:
epoll & kqueue channel implementations have methods starting with write that shouldn't be instrumented